static/main.css

```diff
@@ -20,6 +20,16 @@
   font-family: 'Raleway';
   font-weight: 300;
   src: url("https://storage.googleapis.com/purple-monkey/Raleway/Raleway-Light.ttf") format('truetype');
```

Comment:
Please keep all FutureDaysFarm fonts in the same bucket.  We can revisit the idea of having all projects use the same font bucket in the future, but for now every project should be separate.